src/main/java/frc/robot/commands/VisionTargetDistance.java

```diff
@@ -25,8 +25,12 @@ public VisionTargetDistance(Shooter shooter) {
   @Override
   public void initialize() {
       System.out.println("Getting distance");
-      shooter.setLEDRing(true);
-      SmartDashboard.putString("vision/active_mode/selected", "intake");
+
+      // This is only for testing
+      // Turn on the LED but put the camera into "Intake", which does not find the target!!
+      
+      shooter.vision.setLEDRing(true);
+      shooter.vision.setMode("intake");
```

Comment:
Unfortunately, shooter.vision.setMode("intake") will turn off the LED. Perhaps we need a new mode and an update to setMode?

Reply:
should I edit the method so that the LED also turns on if the mode is "intake"?

Reply:
Trying to remember if/why we would want the LED on in intake mode. I don't think that camera has an LED. We'll have to check with Paul.

Reply:
At this point, just remove line 32. There should be no reason to have the LED on while the camera mode is intake.